docs/extensibility/entity-extension.md

```diff
@@ -18,7 +18,7 @@ This article describes a quick way to extend your entity and the internals of en
         * `User`
     * keep entity and table annotations
     * you can add new properties and use annotations to configure ORM
-* Add information about the entity extension into the container configuration
+* Entities from namespace `App\` extending entities from namespace `Shopsys\` are automatically extended, if you want to extend from or to different namespace you need to add information about the entity extension into the container configuration
     * add it to the configuration parameter `shopsys.entity_extension.map` placed in `config/parameters_common.yml` file
     * use the parent entity name as a key and the extended entity name as a value
     * eg. `Shopsys\FrameworkBundle\Model\Product\Product: App\Model\Product\Product`
```

Comment:
What about to change the example to something that is not covered with compiler pass to be exact?

Reply:
Updated